packages/solana-adapter/src/getCreatePaymentSignedTx.ts

```diff
@@ -0,0 +1,90 @@
+import {
+  Connection,
+  SystemProgram,
+  PublicKey,
+  SYSVAR_RENT_PUBKEY,
+  Signer,
+} from '@solana/web3.js';
+import {
+  ASSOCIATED_TOKEN_PROGRAM_ID,
+  getAssociatedTokenAddress,
+  TOKEN_PROGRAM_ID,
+} from '@solana/spl-token';
+import { AnchorWallet } from '@solana/wallet-adapter-react';
+import { BN, Program } from '@project-serum/anchor';
+import { HelioIdl } from './program';
+import { CreatePaymentStateRequest } from './types';
+import { helioFeeWalletKey, daoFeeWalletKey } from './config';
+import { signTransaction } from './utils';
+
+export const getCreatePaymentSignedTx = async (
+  connection: Connection,
+  wallet: AnchorWallet,
+  program: Program<HelioIdl>,
+  req: CreatePaymentStateRequest,
+  payFees: boolean = true
+): Promise<string> => {
+  const mint = req.mintAddress!;
+
+  const senderAssociatedTokenAddress = await getAssociatedTokenAddress(
+    mint,
+    req.sender
+  );
+
+  const recipientAssociatedTokenAddress = await getAssociatedTokenAddress(
+    mint,
+    req.recipient
+  );
+
+  const paymentAssociatedTokenAddress = await getAssociatedTokenAddress(
+    mint,
+    req.paymentAccount.publicKey
+  );
+
+  const helioFeeTokenAccountAddress = await getAssociatedTokenAddress(
+    mint,
+    helioFeeWalletKey
+  );
+
+  const daoFeeTokenAccountAddress = await getAssociatedTokenAddress(
+    mint,
+    daoFeeWalletKey
+  );
+
+  // eslint-disable-next-line
+  const [_pda, bump] = await PublicKey.findProgramAddress(
```

Comment:
const [, bump] - this way you don't have to disable eslint

Reply:
Ok